tests/tf-manifests/rhcs/clusters/rosa-classic/main.tf

```diff
@@ -45,6 +45,11 @@ locals {
     operator_role_prefix = var.operator_role_prefix
     oidc_config_id       = var.oidc_config_id
   }
+  base_dns_domain      = var.base_dns_domain == null ? null : var.base_dns_domain
```

Comment:
why this check ?

Reply:
It's to be compatible with existing provision process of non-Shared-VPC clusters.
`base_dns_domain` is only used for Shared-VPC cluster.

Reply:
why do we need the check, can't we just use `var.base_dns_domain` directly instead of having a local ?

Reply:
Sorry I misread that, updated.